README.md

```diff
@@ -63,6 +66,52 @@ then run the `osx-setup` target:
     $ make osx-setup JDK=JavaDeveloper.pkg
 
 
+## Build Configuration
+
+The Java.Interop build can be configured by overriding **make**(1) variables
+on the command line or by specifying MSBuild properties to control behavior.
+
+### **make**(1) variables
+
+The following **make**(1) variables may be specified:
+
+* `$(CONFIGURATION)`: The product configuration to build, and corresponds
+    to the `$(Configuration)` MSBuild property when running `$(XBUILD)`.
+    Valid values are `Debug` and `Release`. Default value is `Debug`.
+* `$(RUNTIME)`: The managed runtime to use to execute utilities, tests.
+    Default value is `mono64` if present in `$PATH`, otherwise `mono`.
+* `$(TESTS)`: Which unit tests to execute. Useful in conjunction with the
+    `make run-tests` target:
+
+        make run-tests TESTS=bin/Debug/Java.Interop.Dynamic-Tests.dll
+
+* `$(V)`: If set to a non-empty string, adds `/v:diag` to `$(XBUILD)`
+    invocations.
+* `$(XBUILD)`: The MSBuild build tool to execute for builds.
+    Default value is `xbuild`.
+
+
+### MSBuild Properties
+
+MSbuild properties may be placed into the file `Configuration.Override.props`,
+which can be copied from
+[`Configuration.Override.props.in`](Configuration.Override.props.in).
+The `Configuration.Override.props` file is `<Import/>`ed by
+[`Configuration.props`](Configuration.props); there is no need to `<Import/>`
+it within other project files.
+
+Overridable MSBuild properties include:
+
+* `$(JdkJvmPath)`: Full path name to the JVM native library to link
+    [`java-interop`](src/java-interop) against. By default this is
+    probed for from numerious locations within
+    [`build-tools/scripts/jdk.mk`](build-tools/scripts/jdk.mk).
+* `$(UtilityOutputFullPath)`: Directory to place various utilities such as
+    [`class-parse`](tools/class-parse), [`generator`](tools/generator),
+    and [`logcat-parse`](tools/logcat-parse). By default this is
+    `bin/$(Configuration)`.
```

Comment:
um and accordingly this (`bin/$(Configuration)`) should be updated to show an absolute path or explicitly mention that here.